src/main/scala/cromwell/Main.scala

```diff
@@ -4,20 +4,29 @@ import java.io.File
 import java.nio.file.Paths
 
 import cromwell.binding._
-import cromwell.engine.{WorkflowManagerActor, SingleWorkflowRunnerActor}
 import cromwell.binding.formatter.{AnsiSyntaxHighlighter, SyntaxFormatter}
+import cromwell.engine.SingleWorkflowRunnerActor
 import cromwell.parser.WdlParser.SyntaxError
-import cromwell.server.{WorkflowManagerSystem, CromwellServer}
+import cromwell.server.{CromwellServer, WorkflowManagerSystem}
 import cromwell.util.FileUtil
+import org.slf4j.LoggerFactory
 import spray.json._
 
-import scala.util.{Failure, Success}
-
 object Actions extends Enumeration {
   val Parse, Validate, Highlight, Run, Inputs, Server = Value
 }
 
 object Main extends App {
+  val props = System.getProperties
+  val loggerVar = props.getProperty("CROMWELL_LOGGER")
+  if (loggerVar != null && loggerVar.length == 0) {
+    args.headOption.map {_.capitalize} match {
+      case Some(x) if x == Actions.Server => props.setProperty("CROMWELL_LOGGER", "SERVER")
+      case _ => props.setProperty("CROMWELL_LOGGER", "CONSOLE")
```

Comment:
If I'm reading this right something like this would work:

    args.headOption map {_.capitalize} find {_ == Actions.Server}  match {
         case Some(x) => ???
         case None => ???
    }

Not any different but a bit more idiomatic to use the builtin combinators where appropriate